### ISSUE TITLE
Make index route ending optional

### ISSUE THREAD

**Issue by Meemaw** [toomuchdesign/next-page-tester#22]
## 🚀 Feature Proposal

index ending should be optional in path.

e.g. the following two should be the same:

```ts
const page = await getPage({ route: '/' });
const page = await getPage({ route: '/index' });
```

**Comment by toomuchdesign** [comment]
Oh! Doesn't it work like this, already? If so, this is definitely a bug, Let's file an issue.

Does it happen just for `/index` route, right?

**Comment by toomuchdesign** [comment]
Closed by #24
